Windows Formatting job failing because it can't find Python on the path

I spoke with @MattGal offline and it seems like the new images don't contain python2 anymore and that is what is breaking this job. @MattGal is going to give a shot on fixing this issue. 

```
##[error]Parameter 'toolPath' cannot be null or empty.
```

https://dev.azure.com/dnceng/public/_build/results?buildId=562848

cc: @dotnet/runtime-infrastructure

--- a/eng/pipelines/coreclr/templates/format-job.yml
+++ b/eng/pipelines/coreclr/templates/format-job.yml
@@ -26,7 +26,11 @@ jobs:
     name: ${{ format('format_{0}{1}_{2}', parameters.osGroup, parameters.osSubgroup, parameters.archType) }}
     displayName: ${{ format('Formatting {0}{1} {2}', parameters.osGroup, parameters.osSubgroup, parameters.archType) }}
     helixType: 'format'
-    pool: ${{ parameters.pool }}
+    - ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
+      pool: 
+        vmImage: 'windows-2019'
+    - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+      pool: ${{ parameters.pool }}
     variables: ${{ parameters.variables }}
     condition: ${{ parameters.condition }}
     steps:
@@ -38,6 +42,12 @@ jobs:
         version: '3.x'
         includePreviewVersions: true
         installationPath: $(Agent.ToolsDirectory)/dotnet
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.x' 
+        addToPath: true 
+        architecture: 'x64'
+      condition: ${{ eq(parameters.osGroup, 'Windows_NT') }}
     - task: PythonScript@0
       displayName: Run tests/scripts/format.py
       inputs:

--- a/eng/pipelines/coreclr/templates/format-job.yml
+++ b/eng/pipelines/coreclr/templates/format-job.yml
@@ -26,10 +26,10 @@ jobs:
     name: ${{ format('format_{0}{1}_{2}', parameters.osGroup, parameters.osSubgroup, parameters.archType) }}
     displayName: ${{ format('Formatting {0}{1} {2}', parameters.osGroup, parameters.osSubgroup, parameters.archType) }}
     helixType: 'format'
-    - ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
+    ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
       pool: 
         vmImage: 'windows-2019'
-    - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+    ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
       pool: ${{ parameters.pool }}
     variables: ${{ parameters.variables }}
     condition: ${{ parameters.condition }}

